be/src/exec/tablet_sink.cpp

```diff
@@ -262,6 +279,11 @@ Status NodeChannel::add_chunk(vectorized::Chunk* input, const int64_t* tablet_id
             _cur_chunk = input->clone_empty_with_slot();
         }
 
+        // wait if queue full
```

Comment:
is_full() return true when queue is full and previous request has not finished, which means it can not push data into queue or pop data from queue to send request.
so wait previous request done then pop data from queue to send request.